### PR TITLE
DAC6-2166 : BuisnessType Nunjk transform

### PR DESCRIPTION
**Pull request by AashuChauhanHmrc**

### CODE REVIEW

File: app/views/BusinessTypeView.scala.html [hunk 1]
```diff
@@ -0,0 +1,47 @@
+@*
```
Comment (matthewharris1) [inline app/views/BusinessTypeView.scala.html:1]: Can you please amend the spacing in this file similar to other views

File: app/models/BusinessType.scala [hunk 2]
```diff
@@ -53,6 +56,15 @@ object BusinessType extends Enumerable.Implicits {
     Radios(field, items)
   }
 
+  def options(implicit messages: Messages): Seq[RadioItem] = values.zipWithIndex.map {
```
Comment (matthewharris1) [inline app/models/BusinessType.scala:59]: I can't see what this method is for can you explain?

**Review by matthewharris1 — CHANGES_REQUESTED** [review]

File: app/views/BusinessTypeView.scala.html (hunk 1 above)
Comment (AashuChauhanHmrc) [reply app/views/BusinessTypeView.scala.html:1]: Formatted page as per other view pages.

File: app/models/BusinessType.scala (hunk 2 above)
Comment (AashuChauhanHmrc) [reply app/models/BusinessType.scala:59]: As far as I can understand this method is to identify the businessType based upon the choosen option . Not sure I might be wrong.

**Pull request closed without merging**